templates/terraform/transport.go

```diff
@@ -105,7 +106,16 @@ func sendRequest(config *Config, method, url string, body map[string]interface{}
 		}
 	}
 
-	req, err := http.NewRequest(method, url+"?alt=json", &buf)
+	u, err := urllib.Parse(url)
+	if err != nil {
+		return nil, err
+	}
+
+	queries := u.Query()
+	queries.Add("alt", "json")
+	newUrl := u.Scheme + "://" + u.Host + u.Path + "?" + queries.Encode()
```

Comment:
note to self: https://github.com/GoogleCloudPlatform/magic-modules/pull/191/files does it better